File: js/방 번호(1475).js
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');

const n = input[0].split('');

const card = Array.from({length: 10}, () => 0);

for (let elem of n) {
    if (elem === '6') {
        card[9] += 1;
    }
    else {
        card[Number(elem)] += 1;
    }
}

card[9] = Math.ceil(card[9] / 2);

const maxValue = Math.max(...card);

console.log(maxValue);

File: python/방 번호(1475).py
from collections import Counter

n =list(input())
arr = Counter(n)

arr['9'] += arr['6']
del arr['6']

if arr['9'] % 2 == 0:
    arr['9'] //= 2
else:
    arr['9'] //= 2
    arr['9'] += 1

max_key = max(arr, key=arr.get)
max_value = arr[max_key]

print(max_value)